app/views/welcome_mailer/notify_newcomer.html.erb

```diff
@@ -19,6 +19,9 @@
     Looking forward to hearing from you soon,
 
     The Public Lab Community
+
+    You need to verify your email to confirm your registration. Please click the link below to confirm your email. Note that this link is valid for only 24 hours after its generation. Thank you. 
+    <%="localhost:3000/#{@user.id}/verify/#{@user.generate_token}" %>
```

Comment:
Please check password reset form from other websites and modify it accordingly. Like may be we would prefer a `<a>` tag here on which user can directly click and get account verified ? Also, along with `a` tag, may be there should be text link saying, if you can't click on that link, then copy paste it into the browser ?